iterations_runners/Makefile

```diff
@@ -0,0 +1,26 @@
+HERE != pwd
+C_ITER_RUNNER_CFLAGS = -fPIC -Wall -Wextra -pedantic -g
```

Comment:
Good spot. IIRC, it actually causes minor slowdown (bigger files etc.). I have no idea whether it's a measurable slowdown, though.